pkgs/by-name/sh/shader-slang/package.nix

```diff
@@ -26,15 +26,19 @@
   withSharedLLVM ? withLLVM,
 }:
 
+let
+  minVersionSpirvHeaders = "1.4.305";
+  devendorSpirvHeaders = lib.versionAtLeast spirv-headers.version minVersionSpirvHeaders;
```

Comment:
I am not super cool with this flipping based on other package updates. Do we have better alternatives?

Reply:
We could just decide to permanently prefer the vendored version, as this is the second time in two months that it depends on unreleased changes

Reply:
The other alternative would be to always wait for a new spirv-headers release before updating shader-slang when this happens

Reply:
(my personal preference, if we do remove the conditional, is to stick to the vendored headers)

Reply:
The package now always uses the vendored headers, as I expect Slang to more often than not to require yet unreleased changes to spirv-headers to work. We may revert this in the future once development pace has settled down a bit.